src/app/components/event-card/event-card.component.scss

```diff
@@ -2,58 +2,81 @@
 @import '../../mixins.scss';
 
 .event-card {
-    width: 100%;
-    height: 100%;
-    border: 3px solid variables.$yonder-light-blue;
+    width: 340px;
+    height: 255px;
+    padding: 0;
     box-shadow: 4px 4px 3px #d9d9d9;
-    border-radius: 10px;
+    border-radius: 6px;
 }
 
-.event-image {
+.image-div {
+    justify-content: center;
+    display: flex;
     width: 100%;
+    height: 160px;
+    overflow: hidden;
+    border-radius: 6px 6px 0px 0px;
+}
+
+.event-image {
+    width: 340px;
+    height: 160px;
+    object-fit: cover;
+    object-position: 100% 0;
 }
 
 .about {
-    @include flex-container(column, flex-start);
-    justify-content: space-between;
+    @include flex-container(column, space-between);
+    padding: 0px 8px 8px 8px;
     margin-left: 5px;
     margin-right: 5px;
 }
 
 .first-row {
+    @include flex-container(row, space-between);
     width: 100%;
-    @include flex-container(row, flex-start);
+    margin: auto;
     margin-top: 5px;
-    margin-bottom: 10px;
-    justify-content: space-between;
+    align-items: center;
 }
 
 .title {
-    font-size: 1.6vh;
+    font-size: 20px;
     font-weight: 700;
+    line-height: 110%;
 }
 
 .countdown {
-    @include flex-container(row, flex-start);
+    @include flex-container(row);
+    margin-top: 5px;
+}
+
+.icon {
+    vertical-align: bottom;
+    font-size: 18px !important;
+    margin-right: -5px;
```

Comment:
These are 2 dangereous lines. Can you find another way instead of using !important and '-' on px?
For '!important' you can inspect page and see where's the other font-size that overrides this one and use same path

Reply:
Done